client/__tests__/e2e/config.js

```diff
@@ -1,11 +1,13 @@
-export const jestEnv = process.env.JEST_ENV;
-export const appPort = process.env.CXG_SERVER_PORT;
+// # Similar constants are defined in file: `environment.default`
+export const DEV = "dev";
+export const DEBUG = "debug";
+export const PROD = "prod";
+export const CXG_SERVER_PORT = 5005;
+export const CXG_CLIENT_PORT = 3000;
+export const jestEnv = process.env.JEST_ENV || PROD;
+export const appPort = process.env.CXG_CLIENT_PORT || CXG_CLIENT_PORT;
```

Comment:
See my comment about switching `CXG_CLIENT_PORT` and `CXG_SERVER_PORT` in `client/Makefile`. I think we should avoid this switcheroo.